File: deaf_grandma.py
def deaf_grandma():
    print("HEY KID!")
    counter=0
    while counter <2:
        kid_statement=input()
        if kid_statement=="":
            print("WHAT?!")
        elif kid_statement != kid_statement.upper():
            print("SPEAK UP, KID!")
        elif kid_statement == kid_statement.upper() and kid_statement!= "GOODBYE!":
            print("NO, NOT SINCE 1946!")
        elif kid_statement == "GOODBYE!" and counter <1:
            print("LEAVING SO SOON?")
            counter+=1
        else:
            print("LATER, SKATER!")
            counter+=1


deaf_grandma()

File: deaf_grandma.js
function deafGrandma() {
    let counter=0;
    let kid_statement= window.prompt("HEY KID!")
    while(counter<2){    
        console.log(kid_statement)
        if (kid_statement===""){
            kid_statement= window.prompt("WHAT")
        }
        else if(kid_statement !== kid_statement.toUpperCase()){
            kid_statement= window.prompt("SPEAK UP, KID!")
        }
        else if(kid_statement=== kid_statement.toUpperCase() && kid_statement!=="GOODBYE!"){
            kid_statement= window.prompt("NO, NOT SINCE 1946")
        }
        else if(kid_statement==="GOODBYE!" && counter<1){
            kid_statement= window.prompt("LEAVING SO SOON?")
            counter++
        }
        else{
            alert("LATER, SKATER!")
            counter++
        }
    }
}

deafGrandma();